### DISCUSSION TITLE
Radio in checkout woocommerce

### DISCUSSION THREAD

**Discussion by dimage** [bootscore/bootscore#782] [Q&A]
### Describe the feature that you'd like

Please enter the code on the checkout page woocommerce
How to see this:
![FireShot001](https://github.com/bootscore/bootscore/assets/10204962/9edc1250-a954-4950-8f32-018a41c2ec43)

### Checklist

- [X] I have searched the issues and this feature has not been requested.

**Comment by dimage** [comment]
![FireShot002](https://github.com/bootscore/bootscore/assets/10204962/109bcedd-53ae-4e10-97e6-7ff06dd73ce8)
I get it like this

**Comment by crftwrk** [comment]
Please explain exactly what do you want to do and share a link to your site to check and share your code.

**Comment by dimage** [comment]
I can't provide a link to the site because it is installed locally

I want something like this
![FireShot003](https://github.com/bootscore/bootscore/assets/10204962/195c6393-25e4-4c8e-850f-102abcf46e17)

**Comment by dimage** [comment]
globally I want to make a separation for buyers and companies. show different lists of fields

**Reply by crftwrk** [reply]
Converted this to a discussion, because this is WooCommerce area and has nothing to do with theme. Generally you can use the `woocommerce_before_checkout_billing_form` hook to add something. You can find all available hooks in checkout page [here](https://www.businessbloomer.com/woocommerce-visual-hook-guide-checkout-page/).

Add following snippet to child's `functions-php`:

```php
/**
 * Custom radios to checkout page
 */
add_action( 'woocommerce_before_checkout_billing_form', 'custom_checkout_radios' );

// Function for `woocommerce_before_checkout_billing_form` action-hook.
function custom_checkout_radios(){

  // action...
  echo '
  <div class="mb-3">
    <label class="form-label d-block">Label</label>
    <div class="form-check form-check-inline">
      <input class="form-check-input" type="radio" name="flexRadioDefault" id="flexRadioDefault1" checked>
      <label class="form-check-label" for="flexRadioDefault1">
        Default checked radio
      </label>
    </div>
    <div class="form-check form-check-inline">
      <input class="form-check-input" type="radio" name="flexRadioDefault" id="flexRadioDefault2">
      <label class="form-check-label" for="flexRadioDefault2">
        Default radio
      </label>
    </div>
    <div class="form-check form-check-inline">
      <input class="form-check-input" type="radio" name="flexRadioDefault" id="flexRadioDefault3">
      <label class="form-check-label" for="flexRadioDefault3">
        Default radio
      </label>
    </div>
  </div>
  ';

}
```

Note that this example just adds the radios but have no functionality or submit something to the order. In that case you have to write a custom plugin or install an existing one https://wordpress.org/plugins/custom-checkout-fields-for-woocommerce.
